application/queue/suspend_users.js

```diff
@@ -13,10 +11,15 @@ const config = require('../configuration'),
 
 let deckidsToUserids = {};
 
-if (process.argv[2] === undefined) {
-  console.log('Please provide a JWT!');
-  return;
-}
+// generate a `isReviewer` jwt the deck service will accept  
+const jwt = require('jsonwebtoken');
+const adminToken = jwt.sign({
```

Comment:
Please use controllers/jwt.js for creating tokens.

Reply:
Right, jwt.js in a lib folder would fit better.
You are using a user object in which the email attribute is missing, which is no problem. Using jwt.createToken() with your object should reduce duplicated code.